tmux.conf

```diff
@@ -61,8 +60,8 @@ set -g message-fg colour16
 set -g message-bg colour221
 set -g message-attr bold
 set -g status-left '#[fg=colour235,bg=colour252,bold] ❐ #S #[fg=colour252,bg=colour238,nobold]⮀#[fg=colour245,bg=colour238,bold] #(whoami) #[fg=colour238,bg=colour234,nobold]⮀'
-set -g window-status-format "#[fg=colour235,bg=colour252,bold] #I #W "
-set -g window-status-current-format "#[fg=colour234,bg=colour39]⮀#[fg=black,bg=colour39,noreverse,bold] #{?window_zoomed_flag,#[fg=colour226],} #I: #W #[fg=colour39,bg=colour234,nobold]⮀"
+set -g window-status-format "#[fg=colour235,bg=colour245,bold] #I #W "
```

Comment:
I just like these colors better for the tmux windows stuff. It is gray instead of the bright blue.